test/fixtures/mson/enum-variants.json

```diff
@@ -250,6 +274,15 @@
                       "value": {
                         "element": "enum",
                         "attributes": {
+                          "enumerations": {
+                            "element": "array",
+                            "content": [
+                              {
+                                "element": "string",
+                                "content": "L"
+                              }
+                            ]
```

Comment:
This matches any `string`. Are you missing `fixed` attribute on the `string` elements above?